modules/common/adapters/adapter.h

```diff
@@ -86,9 +86,9 @@ class AdapterBase {
   virtual bool HasReceived() const = 0;
 
   /**
-   * @brief Gets message delay in milliseconds.
+   * @brief Gets message delay.
    */
-  virtual double GetDelayInMs() const = 0;
+  virtual double GetDelay() const = 0;
```

Comment:
can you include the time unit in function name to make it more prominent? As the downstream caller of the function might have different requirement of the time unit, like monitor vs dreamview.